test/player-team.controller.spec.ts

```diff
@@ -34,6 +44,56 @@ describe('palyer-team', () => {
     .body.token;
 
     tokenManager = (await request(`${baseString}`).get('/auth/login').send(manager)).body.token;
+    tokenAdmin = (await request(`${baseString}`).get('/auth/login').send(admin)).body.token;
+
+    const socketAdmin = io.connect('http://localhost:3030', {
+      extraHeaders: {
+        Authorization: `Bearer ${tokenAdmin}`
+      }
+    });
+    const socketManager = io.connect('http://localhost:3030', {
+      extraHeaders: {
+        Authorization: `Bearer ${tokenManager}`
+      }
+    });
+    const socketPlayer = io.connect('http://localhost:3030', {
+      extraHeaders: {
+        Authorization: `Bearer ${tokenNewPlayer}`
+      }
+    });
+
+    socketAdmin.on('connection', function(data) {
+      console.log('connect Admin: ', data);
+    });
+    socketManager.on('connection', function(data) {
+      console.log('connect Manager: ', data);
+    });
+    socketPlayer.on('connection', function(data) {
+      console.log('connect Player: ', data);
+    });
+
+    socketAdmin.on('forAdmin', async function(data) {
+      countAdminResponse++ ;
```

Comment:
push messages into array for each role. with this you'll be able to monitor what were the messages in the test run. You'll be able to clean arrays by reassigning or `.length = 0`